spec/domain/healthchecks/database_connection_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe Healthchecks::DatabaseConnection do
+  it { is_expected.to be_enabled }
+
+  its(:name) { is_expected.to eq(:database_status) }
+
+  describe '#status' do
+    context 'when the connection is valid' do
+      before { Dimensions::Date.create_with(Date.today) }
+
+      its(:status) { is_expected.to eq(:ok) }
+      its(:message) { is_expected.to be_blank }
+    end
+
+    context 'when the connection is not valid' do
```

Comment:
Not clear what conditions create this context?

Reply:
It can be invalid because of the database is down, because it has no available connections, because of an issue with the pool configuration,... 

For the sake of this test, if I can't get the first `Date` of the `Dimensions::Date` I don't have a valid connection. This approach is quite common in Connection Pools when they run a very simple query to validate each connection before making it available. 

I didn't worry too much about the performance of this query because it is actually called every 15 seconds, by the way.

If you can think of a better name for the context, just let me know.

Reply:
Sorry, I meant there isn't anything in the test setup to signify an error in the database connection. We should probably leave a comment explaining that the absence of records is what we are testing.